scalatest/src/main/scala/io/cucumber/scalatest/FeatureSuite.scala

```diff
@@ -0,0 +1,79 @@
+package io.cucumber.scalatest
+
+import io.cucumber.core.exception.CucumberException
+import io.cucumber.core.gherkin.{Feature, Pickle}
+import io.cucumber.core.runtime.CucumberExecutionContext
+import org.scalatest.{Args, Status, Suite}
+
+import java.util.function.Predicate
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+import scala.jdk.OptionConverters.RichOptional
+import scala.util.Try
+
+private[scalatest] final class FeatureSuite(
+    private val feature: Feature,
+    private val uniqueSuffix: Option[Int],
+    private val filter: Predicate[Pickle],
+    private val context: CucumberExecutionContext
+) extends Suite {
+
+  private val children: Seq[PickleSuite] = {
```

Comment:
How many hierarchies can Scala test express? If more than 2, it might be worthwhile to match the layout in the feature file.